### ISSUE TITLE
undefined method `request_sequence' 

### ISSUE THREAD

**Issue by ghost** [ajf8/mcomaster#27]
When I try to run any agent against my machine, I get the following error:
undefined method `request_sequence' for MCollective::Client:Class

I can not seem to find where this method is being called.  When I look at the method, all it does is return a unique ID for the request by bumping up the number by 1.  

Here are the logs:

Started POST "/execute/rpcutil/inventory" for 10.34.230.169 at 2014-10-01 13:34:17 -0400
Processing by ExecuteController#execute as JSON
  Parameters: {"{\"filter\":{\"identity\":"=>{"\"c3osjboss1\""=>{"},\"args\":{}}"=>nil}}, "agent"=>"rpcutil", "mcaction"=>"inventory"}
  ESC[1mESC[35mUser Load (0.2ms)ESC[0m  SELECT  "users".\* FROM "users"  WHERE "users"."id" = 1  ORDER BY "users"."id" ASC LIMIT 1
1aadba02-7ce2-4d80-87c0-31861d333887 Received a request, parameters: {"filter":{"identity":["c3osjboss1"]},"args":{}}
1aadba02-7ce2-4d80-87c0-31861d333887 Converted filters: {"fact"=>[], "cf_class"=>[], "agent"=>[], "identity"=>["c3osjboss1"], "compound"=>[]}
  ESC[1mESC[36m (0.1ms)ESC[0m  ESC[1mbegin transactionESC[0m
  ESC[1mESC[35mSQL (0.3ms)ESC[0m  INSERT INTO "actlogs" ("action", "agent", "args", "created_at", "filters", "owner", "txid", "updated_at") VALUES (?, ?, ?, ?, ?, ?, ?, ?)  [["action", "inventory"], ["agent", "rpcutil"], ["args", "{}"], ["created_at", "2014-10-01 17:34:17.843881"], ["filters", "{\"identity\":[\"c3osjboss1\"]}"], ["owner", "user"], ["txid", "1aadba02-7ce2-4d80-87c0-31861d333887"], ["updated_at", "2014-10-01 17:34:17.843881"]]
  ESC[1mESC[36m (2.3ms)ESC[0m  ESC[1mcommit transactionESC[0m
1aadba02-7ce2-4d80-87c0-31861d333887 Sending acknowledgement.
  ESC[1mESC[35mAppSetting Load (0.1ms)ESC[0m  SELECT  "app_settings".\* FROM "app_settings"  WHERE "app_settings"."set_key" = ? LIMIT 1  [["set_key", "policies_enabled"]]
Completed 200 OK in 8ms (Views: 0.1ms | ActiveRecord: 2.9ms)
  ESC[1mESC[36m (0.0ms)ESC[0m  ESC[1mbegin transactionESC[0m
  ESC[1mESC[35mSQL (0.2ms)ESC[0m  UPDATE "actlogs" SET "mcerr" = ?, "updated_at" = ? WHERE "actlogs"."id" = 6  [["mcerr", "undefined method `request_sequence' for MCollective::Client:Class"], ["updated_at", "2014-10-01 17:34:17.851926"]]
  ESC[1mESC[36m (11.7ms)ESC[0m  ESC[1mcommit transactionESC[0m

**Comment by ajf8** [comment]
Hi there, thank you for the report.

I've seen this in the vagrant environment (are you using it?). It's basically a mismatch between the system mcollective and the gem installed. At the 2.6.0 wasn't on rubygems for mcomaster, but that RPM was coming from the puppetlabs repo, so I tried forcing 2.5.X as the RPM used in Vagrant but that didn't work for some reason.

Now it's in rubygems I've bumped the version in the Gemfile. 

Could you try again please? I've kicked off a rebuilt of the RPM with this.

**Comment by ghost** [comment]
Yeah, I realized that I had a different version installed that had the
method

**Comment by ajf8** [comment]
Cool. Did that fix the issue?

**Comment by ghost** [comment]
Where can I get the RPM?  It seems that mcomaster.org is down, and the yum
repo has not been updated.

**Comment by ghost** [comment]
never mind.  Yum updated it just fine.

**Comment by ghost** [comment]
Yep, updating the version worked like a charm!

Thanks.
